jl4-decision-service/jl4-decision-service.cabal

```diff
@@ -53,11 +53,6 @@ library
     Schema
     Server
 
-  other-modules:
-    Paths_jl4_decision_service
-
-  autogen-modules:
-    Paths_jl4_decision_service
```

Comment:
why was this removed?

Reply:
no more data-files thanks to the nix store method you showed for sourcePaths